README.md

```diff
@@ -16,7 +16,10 @@
 1. Create a token with the `gist` scope and copy it. (https://github.com/settings/tokens/new)
 1. Create a WakaTime account (https://wakatime.com/signup)
 1. In your WakaTime profile settings (https://wakatime.com/settings/profile) ensure `Display coding activity publicly` and `Display languages, editors, operating systems publicly` are checked.
+1. Ensure that username and full name in your waka account is filled out as well.
```

Comment:
Maybe this step is unnecessary since I didn't do any further work but still get the data.